tests/saw/Makefile

```diff
@@ -92,8 +92,9 @@ tmp/%.log : %.saw bitcode/all_llvm.bc tmp
 .PHONY : failure-tests
 failure-tests : bitcode
 	@${MAKE} clean-failure-logs
-	@${MAKE} failure_tests/tls_early_ccs.log
-	@${MAKE} failure_tests/tls_missing_full_handshake.log
+# Temporarily disabling patches that aren't applying correctly
```

Comment:
I know git-blame will bring people to this commit, but let's still give a better comment. When should the patches be re-enabled? Is there an issue we can link to? 

Same for the other disable comments.